### PR TITLE
Allow customization of the package templates

### PR DESCRIPTION
**Pull request by tbetbetbe**
Also
  - update the version (to prepare for a new release)
  - updates the README.md to improve the documentation
  - adds AUTHORS and fixex the LICENSE specifier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,40 +59,51 @@ Optional arguments:
...
   -r REPODIR, --services_root REPODIR
                         The directory containing the rpc service proto
                         definitions. Should not be specified with
-                        -r/--protobuf, which clobbers this.
+                        -u/--zip_url, which clobbers this.
   -u ZIPURL, --zip_url ZIPURL
                         URL of a git repository containing the rpc proto
                         definitions. Should not be specified with
-                        -r/--protobuf_root, which will be ignored
+                        -r/--services_root, which will be ignored
```
Comment (saicheems) [inline README.md]: Consistency nit: some of these descriptions end with periods and some don't.

File: bin/gen-api-package [hunk 2]
```diff
@@ -115,26 +115,26 @@ var parseArgs = function parseArgs() {
...
   );
   cli.addArgument(
     [ '--dependencies_file' ],
     {
-      help: 'Specifies an alternate location for the deps config file\n' +
-        'dependencies configures the package dependencies to include in the ' +
-        ' generated package file',
+      help: 'Specifies an alternate location for the deps config file.\n' +
+        'dependencies.yml configures the per-language dependencies'
+        + ' that each language package relies on',
```
Comment (saicheems) [inline bin/gen-api-package]: Nit: This and all below help strings don't end in periods...

File: bin/gen-api-package [hunk 3]
```diff
@@ -115,26 +115,26 @@ var parseArgs = function parseArgs() {
     [ '--api_defaults_file' ],
     {
       help: 'Specifies an alternate location for the api_defaults config' +
-        ' file\n api_defaults configures shared metadata that is may be' +
-        ' included in all the package files',
+        ' file.\n api_defaults.yml configures shared package metadata to ' +
+        ' be used in packages of any language.',
```
Comment (saicheems) [inline bin/gen-api-package]: Looks like there's a double space here. "to  be"

File: bin/gen-api-package (hunk 2 above)
Comment (saicheems) [inline bin/gen-api-package]: Also, just noticed you use different style all over the file for aligning the string concat - some places you break before/after the + and align with an indent/at the colon.

**Comment by saicheems** [comment]
Content lgtm

File: README.md (hunk 1 above)
Comment (tbetbetbe) [inline README.md]: Fixed

File: bin/gen-api-package (hunk 2 above)
Comment (tbetbetbe) [inline bin/gen-api-package]: Both Fixed

File: bin/gen-api-package (hunk 3 above)
Comment (tbetbetbe) [inline bin/gen-api-package]: Fixed

**Pull request merged**